pkg/operator/target_config_reconciler.go

```diff
@@ -3,6 +3,7 @@ package operator
 import (
 	"context"
 	"fmt"
+	deschedulerv1 "github.com/openshift/cluster-kube-descheduler-operator/pkg/apis/descheduler/v1"
```

Comment:
nit: move this next to other `github.com/openshift/cluster-kube-descheduler-operator` imports